nimble/host/services/audio/bass/include/services/bass/ble_audio_svc_bass.h

```diff
@@ -0,0 +1,174 @@
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef H_BLE_AUDIO_SVC_BASS_
+#define H_BLE_AUDIO_SVC_BASS_
+
+#include <stdint.h>
+#include "host/audio/ble_audio_bsnk.h"
+#include "syscfg/syscfg.h"
+
+#define BLE_SVC_AUDIO_BASS_UUID16                                           0x184F
+#define BLE_SVC_AUDIO_BASS_CHR_UUID16_BAS_CONTROL_POINT                     0x2BC7
+#define BLE_SVC_AUDIO_BASS_CHR_UUID16_BROADCAST_RECEIVE_STATE               0x2BC8
+
+#define BLE_SVC_AUDIO_BASS_CTRL_POINT_EVENT_REMOTE_SCAN_STOPPED             0x00
+#define BLE_SVC_AUDIO_BASS_CTRL_POINT_EVENT_REMOTE_SCAN_STARTED             0x01
+#define BLE_SVC_AUDIO_BASS_CTRL_POINT_EVENT_ADD_SOURCE                      0x02
+#define BLE_SVC_AUDIO_BASS_CTRL_POINT_EVENT_MODIFY_SOURCE                   0x03
+#define BLE_SVC_AUDIO_BASS_CTRL_POINT_EVENT_SET_BROADCAST_CODE              0x04
+#define BLE_SVC_AUDIO_BASS_CTRL_POINT_EVENT_REMOVE_SOURCE                   0x05
+
+enum ble_svc_audio_bass_big_enc {
+    BLE_SVC_AUDIO_BASS_BIG_ENC_NOT_ENCRYPTED,
+    BLE_SVC_AUDIO_BASS_BIG_ENC_BROADCAST_CODE_REQ,
+    BLE_SVC_AUDIO_BASS_BIG_ENC_DECRYPTING,
+    BLE_SVC_AUDIO_BASS_BIG_ENC_BAD_CODE
+};
+
+enum ble_svc_audio_bass_pa_sync {
+    BLE_SVC_AUDIO_BASS_PA_SYNC_DO_NOT_SYNC,
+    BLE_SVC_AUDIO_BASS_PA_SYNC_SYNC_PAST_AVAILABLE,
+    BLE_SVC_AUDIO_BASS_PA_SYNC_SYNC_PAST_NOT_AVAILABLE
+};
+
+enum ble_svc_audio_bass_pa_sync_state {
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_NOT_SYNCED,
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_SYNC_INFO_REQ,
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_SYNCED,
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_SYNCED_FAILED,
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_NO_PAST
+};
+
+struct ble_svc_audio_bass_subgroup {
+    uint32_t bis_sync_state;
+    uint8_t metadata_length;
+    uint8_t *metadata;
+};
+
+struct ble_svc_audio_bass_receiver_state_params {
+    ble_addr_t source_addr;
+    uint8_t source_adv_sid;
+    uint32_t broadcast_id;
+    uint8_t pa_sync_state;
+    uint8_t big_encryption;
+    uint8_t bad_code[BLE_AUDIO_BROADCAST_CODE_SIZE];
+    uint8_t num_subgroups;
+    struct ble_svc_audio_bass_subgroup
+        subgroups[MYNEWT_VAL(BLE_SVC_AUDIO_BASS_MAX_SUBGROUPS_PER_RECV_STATE)];
+};
+
+struct ble_svc_audio_bass_receiver_state {
+    uint8_t source_id;
+    struct ble_audio_bsnk *bsnk;
+    struct ble_svc_audio_bass_receiver_state_params params;
+};
```

Comment:
Probably this can be just
```suggestion
struct ble_svc_audio_bass_receiver_state {
    ble_addr_t source_addr;
    uint8_t source_adv_sid;
    uint32_t broadcast_id;
    uint8_t pa_sync_state;
    uint8_t big_encryption;
    uint8_t bad_code[BLE_AUDIO_BROADCAST_CODE_SIZE];
    uint8_t num_subgroups;
    struct ble_svc_audio_bass_subgroup
        subgroups[MYNEWT_VAL(BLE_SVC_AUDIO_BASS_MAX_SUBGROUPS_PER_RECV_STATE)];
};
```
Because there is no use-case of having `source_id` in public API, as it's just internal thing.